crates/wit-bindgen/src/lib.rs

```diff
@@ -1131,7 +1132,24 @@ impl<'a> InterfaceGenerator<'a> {
                     TypeOwner::None => "<no owner>",
                 },
                 func.name,
-            ));
+            );
+            let mut event_fields = func
+                .params
+                .iter()
+                .enumerate()
+                .map(|(i, (name, _ty))| {
+                    let name = to_rust_ident(&name);
+                    format!("{name} = tracing::field::debug(&arg{i})")
+                })
+                .collect::<Vec<String>>();
+            if event_fields.is_empty() {
+                event_fields.push(format!("\"call\""));
+            }
+            uwrite!(
+                self.src,
+                "tracing::event!(tracing::Level::TRACE, {});\n",
```

Comment:
What do you think about always including `"call"` in the trace?
```suggestion
            uwrite!(
                self.src,
                "tracing::event!(tracing::Level::TRACE, \"call\", {});\n",
```

Reply:
Yeah, I could do this, and always include `"return"` in the other side. It wouldn't hurt. That syntax isnt quite correct, though, so I'll fix it